code/modules/mob/living/carbon/human/species_types.dm

```diff
@@ -45,6 +45,30 @@ datum/species/human/spec_death(gibbed, mob/living/carbon/human/H)
 	if(H)
 		H.endTailWag()
 
+//Inspired by the Syndicate agents of Bullfrog's Syndicate masterpiece
+//The corporate post-human
+/datum/species/corporate
+	name = "Corporate Agent"
+	id = "agent"
+	hair_alpha = 0//The procedure left them bald
+	need_nutrition = 0//They don't need to eat
+	say_mod = "asserts"//Their word is law
+	speedmod = 1//The cybernetic limbs cost a fortune
+	brutemod = 0.5//and it cost them their humanity
+	burnmod = 0.5//but they were worth everything
+	coldmod = 0
+	heatmod = 0//they can walk through a plasmafire, naked, unphased
+	punchdamagelow = 20
+	punchdamagehigh = 50//they are inhumanly strong
+	punchstunthreshold = 100
```

Comment:
That's kind of the point, they don't really NEED to stun when their fists are basically short range pulse rifles with a no ammo.